btc/src/main/kotlin/helper/network/BitcoinNetworkHelper.kt

```diff
@@ -15,9 +15,12 @@ private val logger = KLogging().logger
 /**
  * Starts bitcoin blockchain downloading process
  */
-fun startChainDownload(peerGroup: PeerGroup, host: String) {
+fun startChainDownload(peerGroup: PeerGroup, hosts: List<String>) {
     logger.info { "Start bitcoin blockchain download" }
-    peerGroup.addAddress(InetAddress.getByName(host))
+    hosts.forEach { host ->
```

Comment:
Should we foresee something like `hosts.filter { host -> !host.isEmpty() }.forEach {` ?

Reply:
I don't think so. Host will be validated by `InetAddress.getByName()`